angular-client/src/pages/charging-page/components/cell-temp/cell-temp-display/cell-temp-display.component.ts

```diff
@@ -11,22 +11,30 @@ import { floatPipe } from 'src/utils/pipes.utils';
 export default class CellTempDisplay {
   avgTemp: number = 0;
   maxTemp: number = 0;
+  resetGraph: boolean = false;
+  resetGraphButton = {
+    onClick: () => {
+      this.resetGraph = true;
+    },
+    icon: 'restart_alt'
+  };
+  mobileThreshold = 1000;
+  isDesktop = window.innerWidth > this.mobileThreshold;
 
-  // -------- Commented out for now, until mobile view is implemented -----//
-  // mobileThreshold = 1000;
-  // isDesktop = window.innerWidth > this.mobileThreshold;
-  // @HostListener('window:resize', ['$event'])
-  // onResize() {
-  //   this.isDesktop = window.innerWidth >= this.mobileThreshold;
-  // }
+  @HostListener('window:resize', ['$event'])
+  onResize() {
+    this.isDesktop = window.innerWidth >= this.mobileThreshold;
+  }
 
   constructor(private storage: Storage) {}
 
   ngOnInit() {
     this.storage.get(IdentifierDataType.CELL_TEMP_HIGH).subscribe((value) => {
+      this.resetGraph = false; //!!!! FOR REVIEW !!!!: is there a better way to do this reseting of the graph boolean?
```

Comment:
you should store the cell temp data as an attribute in this component and then pass it down as prop to the graph. That way you can just clear the array whenever the button is pressed, then you shouldn't need resetGraph